src/drc/api/tests/test_dso_api_strategy.py

```diff
@@ -39,6 +39,7 @@ def assertErrorResponse(self, view, expected_data: dict):
 
         expected_status = expected_data['status']
         self.assertEqual(response.status_code, expected_status)
+        self.assertEqual(response['Content-Type'], 'application/error+json')
```

Comment:
Moet het niet zijn `application/problem+json` ?

Reply:
euh ja... krijg je als je afgeleid bent :-)

Reply:
Ik zal dan niet overal dezelfde opmerking plaatsen over de verschillende componenten :)

Reply:
Nee, is een constante aanpassen in `gemma-zaken-common`, dus andere tests gaan dan ook falen ;)